src/store/ducks/generic/structures.ts

```diff
@@ -103,7 +106,7 @@ export const fetchGenericStructures = (
         return structure;
       }
     ),
-    'id'
+    'structure_id'
```

Comment:
Should we really make this change?  Won't it affect things like IRS reporting?

Reply:
This is because it is possible to have the same structure id present in different plans.  That is why we have another field, `id`, that is absolutely unique